src/main/java/org/sagebionetworks/bridge/dynamodb/DynamoDataPipelineHelper.java

```diff
@@ -0,0 +1,178 @@
+package org.sagebionetworks.bridge.dynamodb;
+
+import com.amazonaws.regions.Region;
+import com.amazonaws.services.datapipeline.model.Field;
+import com.amazonaws.services.datapipeline.model.PipelineObject;
+import com.amazonaws.services.dynamodbv2.model.TableDescription;
+import com.google.common.collect.Lists;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+import java.time.ZoneId;
+import java.time.ZoneOffset;
+import java.time.ZonedDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.temporal.ChronoUnit;
```

Comment:
I heard these new classes in Java 8 were inspired by Joda time. I haven't looked into it so I'm looking around now, here's an (old-ish) blog post that seems to recommend switching to java.time http://blog.joda.org/2014/11/converting-from-joda-time-to-javatime.html
http://blog.joda.org/2009/11/why-jsr-310-isn-joda-time_4941.html